.github/workflows/AssetsUpdater.yml

```diff
@@ -0,0 +1,37 @@
+name: Assets Updater
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**.ts"
+      - "**.json"
+jobs:
+  update:
+    if: github.repository_owner == 'PreMiD'
+    name: Update Assets
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node.js 16
+        uses: actions/setup-node@v3.5.1
+        with:
+          node-version: 16
+          cache: npm
```

Comment:
Maybe we should update to LTS at this point?

Reply:
Copied from other workflows we have, we should change at a later point